### ISSUE TITLE
Malloy-py is not available on pypi

### ISSUE THREAD

**Issue by blackrez** [malloydata/malloy-py#1]
Malloy-py is not available on pypi. 
For install the alpha, installation throught github works.
```
pip install git+https://github.com/malloydata/malloy-py
```

**Comment by gmaden** [comment]
That is correct, the python package is currently still only available on [test.pypi](https://test.pypi.org/project/malloy/2022.1006a0/) 
In the meantime if you wish to install through pip, you can specify test.pypi.org as an index to like the command below: 

```
python3 -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ malloy==2022.1006a0
```
